RZViewActions/UIView+RZViewActions.m

```diff
@@ -128,6 +142,17 @@ - (void)_runWithCompletion:(RZViewActionCompletion)completion
 
 @end
 
+#pragma mark - RZViewSpringAction
+
+@implementation RZViewSpringAction
+
+- (void)_runWithCompletion:(RZViewActionCompletion)completion
```

Comment:
Lame. Is it stated anywhere that they're reserved by Apple, or is that just the consensus?